tripal_chado/src/Plugin/TripalStorage/ChadoStorage.php

```diff
@@ -1257,7 +1320,7 @@ static public function drupalEntityIdLookupCallback($context) {
 
     // During publish, record_id may be null. In this particular case, return null.
     if (!$record_id) {
```

Comment:
```suggestion
    if (!$record_id) {
```
If this is no longer a special case, the special case comment can be dropped.